src/lib/mina_base/account_update.ml

```diff
@@ -1538,6 +1538,10 @@ include T
 let account_id (t : t) : Account_id.t =
   Account_id.create t.body.public_key t.body.token_id
 
+let verification_key (t : t) :
```

Comment:
the function name suggests it's pulling out the vk, but it's converting it to an option.

maybe `verification_key_update_to_option` ...